src/polycubed/src/rest_server.h

```diff
@@ -52,7 +52,9 @@ class RestServer {
 
   static const std::string base;
 
-  void init(size_t thr = 1, const std::string &server_cert = "",
+  void init(size_t thr = 1,
+            size_t max_payload_size = 64*1024,
```

Comment:
Can you please add a comment that says how big is the payload? Just because you told me "I set the max payload to 1MB", but from the code it looks like 64KB, hence it looks confusing to me.

Reply:
64KB is the default init value in the rest_server.h header;
The 1MB setting is in [polycubed.cpp](https://github.com/polycube-network/polycube/blob/473f505c4f252e98473ea97a83bf9a17fd5166e1/src/polycubed/src/polycubed.cpp#L277), where the rest server is being initialized with custom settings